test/integration/integration_test.go

```diff
@@ -0,0 +1,25 @@
+package integration_test
+
+import (
+	"io/fs"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/rogpeppe/go-internal/testscript"
+	"github.com/stretchr/testify/require"
+)
+
+func Test(t *testing.T) {
+	testscript.Run(t, testscript.Params{
+		Dir: "script",
+		Setup: func(env *testscript.Env) error {
+			// Copy the wasm binary into the testscript work directory
```

Comment:
testcript excels at files you can define inline in the txtar scripts. However, the wasm binary needs to be moved to the work directory manually since we can't include it in the txtar file.

Reply:
removed the use of the testscript library to keep the sdk dependencies trim